Maddy-maddigordon757/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -10,3 +10,7 @@
   conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
 */
 
+for (var i = 99; i>0; i--){
+  b=i-1
```

Comment:
It's great that you created a second variable storing the n-1 value you'll be using. Remember that we want our variable names to be as descriptive as possible, so you might want to go with something like "newBottles" here.